### DISCUSSION TITLE
Discovered a bug: mapping to the public network through a firewall, remotely accessing from the public network to both inside and outside, will prompt to be disabled/发现一个BUG：通过防火墙映射到公网，从公网远程到内外，会提示被关闭

### DISCUSSION THREAD

**Discussion by tangjunnan** [rustdesk/rustdesk-server#346] [Show and tell]
I am using version 1.1.9 to map to the public network through a firewall, with the aim of allowing users from the public network to remotely connect to the internal network. However, this cannot be achieved during the actual process and will prompt to be disabled. The system log is shown in the following figure

It feels like this is a bug that requires direct communication between both ends for remote desktop connection. Dear author, please fix this issue in future versions,
Both versions 1.1.8-2 and 1.1.9 have this issue

Chinese version
我使用1.1.9的版本通过防火墙映射到公网，目的是为了让公网的用户远程连接到内网，在实际过程中无法实现，会提示被关闭，系统日志如下图
感觉这是一个BUG，好像需要两端能直接通信才能进行远程桌面链接，请亲爱的作者在后面的版本中修复这个问题啊，
1.1.8-2和1.1.9的版本都存在这个问题

![image](https://github.com/rustdesk/rustdesk-server/assets/157594919/f5c5137f-679d-4bd8-9c06-266524f5399a)

**Comment by 3329415183** [comment]
我现在遇到了一个类似问题，我有几十个Windows设备 突然间有个别设备(至少10个)在发起连接时hbbr的log提示closed 客户端则提示连接超时 我无法理解这到底是什么样的问题 防火墙配置了any to any 或者是关闭都无法解决这个问题，我更倾向是Relay的问题
<img width="1489" alt="image" src="https://github.com/rustdesk/rustdesk-server/assets/61672766/5aa07075-49b3-4779-b9ca-c62440f44cf2">
